content/assignments/techjobs-mvc/getting-started/_index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Getting to Know the Starter Code"
+date: 2023-05-22T10:20:54-05:00
+draft: false
+weight: 1
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Getting Started
+
+Set up a local copy of the project:
+
+<!-- TODO: Add internal link back to Assignment 2 -->
+1. In Canvas, **Graded Assignment #3: TechJobs (MVC Edition)** contains a GitHub repo link. Fork, clone, and then set up the project in IntelliJ. Refer back to the instructions from [Assignment 0]({{< relref "../../hello-world/#getting-started">}}) for details. 
```

Comment:
Suggestion - go to actions tab on the repo and accept/ enable them for autograding.

Reply:
I linked to the place where that is highlighted in Assignment 0 so it should be clear! Thanks for the suggestion! I will definitely ask staff for the first course to keep an eye out for it in case it isn't.